### DISCUSSION TITLE
Seeking Help with Unhandled Error in Next.js Project

### DISCUSSION THREAD

**Discussion by SMGolamAlif** [community/community#83029] [Programming Help]
### Body

I am encountering a persistent error in my Next.js project and would appreciate community assistance in resolving it. The error message is as follows:
```bash 
1 of 1 unhandled error
Server Error
TypeError: mod is not a function
````
Call Stack:

<unknown>
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/swiper/shared/swiper-core.mjs (3321:7)
Array.forEach
<anonymous>
new Swiper
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/swiper/shared/swiper-core.mjs (3320:20)
initSwiper
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/swiper/swiper-react.mjs (178:25)
<unknown>
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/swiper/swiper-react.mjs (192:5)
renderWithHooks
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (5658:16)
renderForwardRef
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (5842:18)
renderElement
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (6005:11)
renderNodeDestructiveImpl
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (6104:11)
renderNodeDestructive
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (6076:14)
renderNode
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (6259:12)
renderChildrenArray
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (6211:7)
renderNodeDestructiveImpl
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (6141:7)
renderNodeDestructive
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (6076:14)
renderNode
file:///home/g_alif/Monzuara%20Fashion-%20E.com/node_modules/react-dom/cjs/react-dom-server.browser.development.js (6259:12)

Efforts to Resolve:
- Updated project libraries to the latest versions.
- Checked release notes for breaking changes.
- Rolled back to previous library versions.
- Verified compatibility of peer dependencies.
- Ensured compatibility between Swiper and Swiper React.
- Cleared Yarn's global cache and reinstalled dependencies.

Any guidance or suggestions from the community would be highly appreciated. Thank you!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by varunpats** [comment]
Hey @SMGolamAlif, [try this](https://stackoverflow.com/questions/77027681/typeerror-mod-is-not-a-function-error-in-reactjs). Import dependencies as per the latest syntax.

**Reply by SMGolamAlif** [reply]
Unfortunately, the suggested solution didn't work for me. I've already structured my code in that manner, but it's still not functioning as expected. Appreciate your assistance.

Here's the relevant part of my code:
````bash 
import { useRef } from "react";
import { IProduct } from "boundless-api-client";
import SliderProductItem from "./productsSlider/SliderProductItem";
import ProductItemLoader from "./productsSlider/ProductItemLoader";
import "swiper/swiper-bundle.css";
import { SwiperProps } from "swiper";
import clsx from "clsx";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faChevronLeft } from "@fortawesome/free-solid-svg-icons/faChevronLeft";
import { faChevronRight } from "@fortawesome/free-solid-svg-icons/faChevronRight";
import { Swiper, SwiperSlide } from "swiper/react";
import SwiperCore, { Navigation, Scrollbar } from "swiper/core";

// Swiper version: 11.0.5
// Repository Link: https://github.com/SMGolamAlif/Monzuara-Fashion-store
````
If you have any further suggestions or insights, I would greatly appreciate it. Thank you!

Rrepositore link : https://github.com/SMGolamAlif/Monzuara-Fashion-store

**Comment by ndpaslam** [comment]
Response #2

**Reply by SMGolamAlif** [reply]
I didn't get it.
